src/main/java/org/example/AccountService.java

```diff
@@ -72,15 +70,10 @@ public void withdrawAmount(Session session){
         System.out.print("Enter Amount to Withdraw: ");
         float withdrawAmount = sc.nextFloat();
         float money = amount-withdrawAmount;
-        String val = "-"+String.valueOf(withdrawAmount);
         if(type==AccountType.SAVING && money>=5000){
-            account.setBalance(money);
-            transcation.setAmount(val);
-            session.save(transcation);
+            setAccountService(money, -withdrawAmount, transcation, account, session);
```

Comment:
its not good to have like -withdrawAmount
you should do like this

float withdrawAmount = sc.nextFloat();
withdrawAmount = withdrawAmount * -1;
then use withdrawAmount (it has negative value now)